channelmonitor/channelmonitor.go

```diff
@@ -433,17 +458,17 @@ func (mc *monitoredChannel) sendRestartMessage(restartCount int) error {
 	// Note that at the networking layer there is logic to retry if a network
 	// connection cannot be established, so this may take some time.
 	p := mc.chid.OtherParty(mc.mgr.PeerID())
-	log.Infof("%s: re-establishing connection to %s", mc.chid, p)
+	log.Debugf("%s: re-establishing connection to %s", mc.chid, p)
 	start := time.Now()
 	err := mc.mgr.ConnectTo(mc.ctx, p)
 	if err != nil {
 		return xerrors.Errorf("%s: failed to reconnect to peer %s after %s: %w",
 			mc.chid, p, time.Since(start), err)
 	}
-	log.Infof("%s: re-established connection to %s in %s", mc.chid, p, time.Since(start))
+	log.Debugf("%s: re-established connection to %s in %s", mc.chid, p, time.Since(start))
 
 	// Send a restart message for the channel
-	log.Infof("%s: sending restart message to %s (%d consecutive restarts)", mc.chid, p, restartCount)
+	log.Debugf("%s: sending restart message to %s (%d consecutive restarts)", mc.chid, p, restartCount)
```

Comment:
@dirkmc We need to log the fact that we saw a disconnect and that we've successfully sent a Restart at the Info level. Do we do that anywhere ?

Reply:
Yes on line 451: https://github.com/filecoin-project/go-data-transfer/pull/241/files/e19701052b4d904efe3efb9d7ec1d86545e133af#diff-20982191cb1617ba3ed900d5e26a716e2682ff63cfc53791f3c4feb34927e1c8R451